WalletWasabi/Crypto/UnblindedSignature.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using NBitcoin.DataEncoders;
+using NBitcoin.Secp256k1;
+
+namespace WalletWasabi.Crypto
+{
+#nullable enable
```

Comment:
We should add `#nullable restore` at the end of the class.

Reply:
Not necessary